### DISCUSSION TITLE
Is it possible to have the cursor change to a pointer when hovering on a particle?

### DISCUSSION THREAD

**Discussion by jeffmanassa97** [matteobruni/tsparticles#5248] [Q&A] [answered]
I am creating a particle menu with the menu items being particles, which will have click events. 
Some info on my implementation -> In the Particles "loaded" prop, I've added custom particles with unique IDs (particle.data.id) to the container and added a container.addClickHandler to handle page re-direction for each particle click based on particle IDs.
Is it possible to have the cursor change to a pointer when hovering on a particle to help denote that they are clickable? Is there a hoverHandler similar to container.addClickHandler that I can add this cursor behavior?

**Comment by matteobruni** [comment, marked as answer]
> I am creating a particle menu with the menu items being particles, which will have click events. Some info on my implementation -> In the Particles "loaded" prop, I've added custom particles with unique IDs (particle.data.id) to the container and added a container.addClickHandler to handle page re-direction for each particle click based on particle IDs. Is it possible to have the cursor change to a pointer when hovering on a particle to help denote that they are clickable? Is there a hoverHandler similar to container.addClickHandler that I can add this cursor behavior?

There is no such behavior out of the box, maybe you can achieve it with a custom plugin like the existing external interactors.
